src/msw/graphicsd2d.cpp

```diff
@@ -1772,9 +1772,12 @@ void wxD2DPathData::AddPath(const wxGraphicsPathData* path)
     GeometryStateData curStateSrc;
     pathSrc->SaveGeometryState(curStateSrc);
 
+    // I think this assert is incorrect. We need to support paths where the end
+    // is not connected to the beginning. --Robin
     // Raise warning if appended path has an open non-empty sub-path.
-    wxASSERT_MSG( pathSrc->IsStateSafeForFlush(),
-        wxS("Sub-path in appended path should be closed prior to this operation") );
+    // wxASSERT_MSG( pathSrc->IsStateSafeForFlush(),
```

Comment:
Let's remove it if it's really not needed, leaving it commented out is not really useful. OTOH I'm not completely sure it's not needed... What is the example of a path for which it is wrongly triggered?

@a-wi could you please look at this?

Reply:
Here is an example:

![Snap001](https://user-images.githubusercontent.com/808502/64360908-4cc28e00-cfc0-11e9-85e9-42efc790dfb5.png)

If this path is closed then a line will be drawn from the trailing end to the beginning of the path, which is not correct in this case.

Reply:
Here is the SVG file for the above image if anybody is curious. GitHub won't let me upload the file, but it's short so I'll just paste it.

```
<?xml version="1.0" encoding="utf-8"?>
<svg viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg">
  <polyline style="fill: none; stroke-width: 3px; stroke: rgb(0, 21, 255); stroke-linecap: round; stroke-linejoin: round;" points="26.576 22.775 177.684 22.775 177.684 58.695 25.673 58.695 25.673 88.195 173.737 88.195 173.737 129.814 25.764 129.814 25.764 169.099 175.947 169.099"/>
</svg>
```

Reply:
I'm going to go ahead and remove the commented out asserts. Let's discuss on wx-dev if it's not correct.